tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -11,9 +11,10 @@
 <pdm_config.conf>
 <tokens.conf>
 <virtual.conf>
-<passthrough-playback.conf>
-<passthrough-capture.conf>
```

Comment:
Please don't remove passthrough-capture.conf, dmic-generic.conf will need it.
cavs-nocodec,conf includes platform/intel/dmic-generic.conf

Reply:
sure, thanks